corehq/apps/app_manager/models.py

```diff
@@ -1855,6 +1855,18 @@ class AdvancedForm(IndexedFormBase, NavMenuItemMediaMixin):
     actions = SchemaProperty(AdvancedFormActions)
     schedule = SchemaProperty(FormSchedule, default=None)
 
+    @classmethod
+    def wrap(cls, data):
+        # lazy migration to swap keys with values in action preload dict.
+        # http://manage.dimagi.com/default.asp?162213
+        load_actions = data.get('actions', {}).get('load_update_cases', [])
+        for action in load_actions:
+            preload = action['preload']
+            if preload and preload.values()[0].startswith('/'):
+                action['preload'] = {v: k for k, v in preload.items()}
```

Comment:
this is safe since there is already validation in the UI that prevents duplicate values